persist/fs/manager.go

```diff
@@ -82,21 +82,21 @@ func (pm *persistManager) persist(
 	rateLimitMbps := pm.rateLimitOpts.LimitMbps()
 	if pm.rateLimitOpts.LimitEnabled() && rateLimitMbps > 0.0 {
```

Comment:
Is it worth caching the limitmbps, limitenabled and limitcheckevery since we perform this once per series, per time window? i.e. MMs * 24